src/main/java/com/samoilenko/onlinebookstore/service/impl/BookServiceImpl.java

```diff
@@ -37,4 +37,23 @@ public BookDto findBookById(Long id) {
         ));
     }
 
+    @Override
+    public void deleteById(Long id) {
+        idValidation(id);
+        bookRepository.deleteById(id);
+    }
+
+    public BookDto update(Long id, BookRequestDto bookRequestDto) {
+        idValidation(id);
+        Book updatedBook = bookMapper.toModel(bookRequestDto);
+        updatedBook.setId(id);
+        return bookMapper.toDto(bookRepository.save(updatedBook));
+    }
+
+    private void idValidation(Long id) {
```

Comment:
```suggestion
    private void validateId(Long id) {
```
method name is almost always an action